src/components/ProfileForm/components/ProfileFormStep2.tsx

```diff
@@ -108,126 +93,146 @@ const ProfileFormStep2 = ({
           <legend className="mb-2 font-bold"> What products you use? </legend>
           <ProfileFormLabelledInput
```

Comment:
Same as the file above.

NOTE: please in the future pay attention to situations like this. If you ever have a UI where you render the same component multiple times, like you do here:

```
<ProfileFormLabelledInput />
<ProfileFormLabelledInput />
<ProfileFormLabelledInput />
...
```

think if you can put the data in an Array and MAP over it.